bootstrap-joining-demo/local/src/main/scala/akka/cluster/bootstrap/Main.scala

```diff
@@ -0,0 +1,19 @@
+package akka.cluster.bootstrap
+
+import akka.actor.ActorSystem
+import akka.management.AkkaManagement
+import akka.management.cluster.bootstrap.ClusterBootstrap
+
+object Main extends App {
+
+  val system = ActorSystem()
+
+  //#akka-management
+  AkkaManagement(system).start()
+  //#akka-management
+
+  //#bootstrap
+  ClusterBootstrap(system).start()
```

Comment:
Are both AkkaManagement and ClusterBootstrap `start` needed?

can't ClusterBootstrap also start AkkaManagement

then probably make them idempotent so that it can only be started once

Reply:
You can also add the akka management routes to your own rather than have akka management start a http server